cliquet/storage/memory.py

```diff
@@ -201,6 +201,7 @@ def create(self, collection_id, parent_id, record, id_generator=None,
         self.set_record_timestamp(collection_id, parent_id, record,
                                   modified_field=modified_field)
         self._store[collection_id][parent_id][_id] = record
+        self._cemetery[collection_id][parent_id].pop(_id, None)
```

Comment:
I used `pop()` as `delete key if exist`. Should I `delete` with `try/except KeyError` instead ?